free-programming-books-tr.md

```diff
@@ -26,18 +26,22 @@
 ### Android
 
 * [Android Geleceği Yazanlar](https://gelecegiyazanlar.turkcell.com.tr/konu/android)
+* [Android Dersleri] (http://umiitkose.com/android/)
 
 
 ### C
 
 * [C Programlama Diline Giriş](http://www1.gantep.edu.tr/~bingul/c/index.php) - [Ahmet Bingül](http://www1.gantep.edu.tr/~bingul)
 * [Programlamaya Giriş Ders Notları](http://web.itu.edu.tr/uyar/programlama) - [H.Turgut Uyar](http://web.itu.edu.tr/uyar)
+* [C Programlama Dersleri](https://www.youtube.com/watch?v=FtaKEn2f2qI&list=PLfzhcDNz4tlXOvEE7z_u3gEqFO3nwIZ5i&index=1)
```

Comment:
This should belong  in a different file location "free-courses-tr.md" since it is a Youtube video and not a book. 

Also, since that file does not exist yet, you are able to create a new file and add the video there!